### ISSUE TITLE
Multiple Marketplace 

### ISSUE THREAD

**Issue by iamsr** [amz-tools/amazon-sp-api#122]
In amazon documentation, it's stated that we just have to connect one marketplace of that region and all other marketplaces of that region will be accessible by that, this works for the UK when I connect with the UK I got all EU regions (who have EU in there amazon seller central URL) but when I connect JAPAN for FE it connected only japan in getMarketplaceParticipation API it returns only japan, not Australia, etc.

**Comment by amz-tools** [comment]
Hi @iamsr,

could it be possible that the seller account is only registered in Japan and does not participate in the other FE marketplaces?

**Comment by iamsr** [comment]
No, it is registered in another marketplace, I can add a listing in Australia but can't access those listing when I am connected with the JAPAN marketplace. If I connect with Australia then I am able to get Australia listing data in calls but then it's only 1 marketplace at a time not another marketplace of that region for FE.

**Comment by amz-tools** [comment]
@iamsr, thats interesting. We have never used the FE endpoint, so I can't really check if that's an account issue or if this happens with all accounts in FE. But that might be a good question to ask the Amazon support.

**Comment by wangjue666** [comment]
@iamsr yes, Countries in the Fast East site require separate authorization

**Comment by cohlar** [comment]
Amazon generates separate refresh tokens for Japan, Australia and Singapore (Far East endpoint) and the UAE (Europe endpoint) - see screenshot below.
I can't find much about it in the [SPAPI documentation](https://developer-docs.amazon.com/sp-api), I found out when opening a new Seller Central account - I would assume it's similar for Egypt, Turkey, India (Europe endpoint) and Brazil (North America endpoint).

![image](https://github.com/amz-tools/amazon-sp-api/assets/57757008/75001781-518d-4ffd-a452-37d5f40b3627)

**Comment by amz-tools** [comment]
I think this has been already answered by @wangjue666  and @cohlar. Closing this.
